src/status_im/chat/events.cljs

```diff
@@ -2,6 +2,7 @@
   (:require [re-frame.core :as re-frame]
             [taoensso.timbre :as log]
             [status-im.utils.handlers :as handlers]
+            [status-im.utils.gfycat.core :as gfycat-utils]
```

Comment:
why not just gfycat ?

Reply:
Just because it's in utils upper level namespace, but no I have no strong feeling about that, I can revert it to `gfycat` :)

Reply:
yes please or utils.gfycat but no new symbol